agent/test_data/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
I think it would be better to put the following `.gitignore` in the agent directory.

```
/test_data
```

Then, create `test_data` directory in BeforeSuite.